docs/setup/install.md

```diff
@@ -1,85 +1,236 @@
 # Installing Open Angel Arena
```

Comment:
I've been nitpicking about little technical shit and I'm sorry for wasting your time. But actually having sat down to read it from start to finish just now, i didn't get past the first page without spotting several continuity errors. I have no idea what this document is supposed to do other than confuse the fuck out of people.

No "newb" is going to get any value out of this document.

This shit needs a full rewrite. Again, I'm sorry for wasting your time.

Reply:
the errors are so pervasive I'm almost convinced that you simply copy pasted it piecemeal from a textbook (which would be plagiarism). 

i understand why you've ordered the sections (required content, github basics, etc) the way you have. You have to cover things in the proper order, you need the software, then the github a ccount, then you can install.

however, you've mentioned things under "Github" that no user could possibly understand without having already read a later section. You may not see this as a problem because "it contains the info [somewhere] " right? But it's a huge problem. You've titled one of your sections "What the heck is a remote?" When there was no previous mention of a remote in the text for them to ever ask such a question. And the resulting explanation of "what a remote is" does not give the user the required context to understand why they just read it, what it means for them, or whether or not the info is useful at all.

In the Section about "github" you start off with an explanation of how to fork a repository. Great, perfect.

Then you go on to explain pull requests... um that's a bit abrupt. they've just learned to create a fork and already they're being told "[Pull requests are] the only way to make changes to Open Angel Arena." Yeah that's goign to confuse them. They will assume this info is being told to them in a certain order for a certain reason.

I think actually the problem can be solved easily. No need for a rewrite. Ask me how!

Reply:
What you need to do before I approve this document:

1. Remove "Github (website)" and "Github (client)" from the document *completely*. They do not add to the document. they merely confuse the reader.

2. Make sure the document reads properly and completely without those two sections (no references to text that is no longer there, etc)

3. Submit only the "required software", "installation", and "manual installation" sections to the "setup" folder

4. Submit the Github stuff to the... wait for it... Github folder! Two documents! Not one! Two!

That solves everything!!!!

Reply:
How do you want me to approach the github for noobs piece?  Simply replace the sections that I wrote pieces for?  The existing document goes into parts I didn't cover, meaning we should try not to do a complete replace.

Reply:
just however you can combine it i suppose

Reply:
I've been working on this on and off for several hours now.  I think the installation docs are in a better place with the removal, but the tradeoff is that the GitHub for Noobs docs feels like a hodge podge of several people with no regard for what was originally there.

Setup:
https://github.com/IBurn36360/oaa/blob/oaa-i-b-s/docs/setup/install.md

GitHub for Noobs:
https://github.com/IBurn36360/oaa/blob/oaa-i-b-s/docs/git/github-for-noobs.md

I'm considering doing an organizational pass completely on the GitHub for Noobs doc in its entirety in order to make it feel better.

Reply:
What I can do, is split this into 2 parts, one would be the initial installation docs, the other would be an organizational pass on the GitHub for Noobs doc in its entirety.  If that seems reasonable, we could get the first part out as soon as tonight.  If it should be a single larger PR, thet might take a night or more.

Reply:
Yeah this is fine for the install doc. Do another PR for the "Github for noobs" piece. It is still very high priority but I don't see why we can't just put this "setup guide" in exactly like it is and then do the github in a different PR